### DISCUSSION TITLE
Import vbp - amazing

### DISCUSSION THREAD

**Discussion by CJCK** [twinbasic/twinbasic#1185] [General]
I am absolutely amazed that beta 124 can actual open our large commercial vb6 program ! What an amazing achievement from a small development team ! As can be expected, there are numerous errors, but based on a quick review, almost all the issues are related to a fairly small number of missing features (less than a dozen I would guess). 

My sincerest thanks to Wayne and others who have helped to achieve this milestone. Please keep up the good work.

cjck

**Comment by dmrvb** [comment]
I also have just imported my biggest and most demanding project into tB. It reported over 1000 errors but it worked, I also am impressed and excited. Its all starting to feel very close now, don't give up Wayne.  Most of the forms look pretty much how they should
I won't report many of the errors as bugs just yet as I suspect at lot will be fixed in the next couple of weeks.
But just for the record:...
Most errors are due to the incomplete implementation of Control Arrays
also
Load Me does not work
ImageBox.borderstyle is read only
There is no Control type. (dim ctr as Control)
There is no DataObject, but then dragdrop is not yet implemented.
and a few more but thats really not a very big list.

**Reply by WaynePhillipsEA** [reply]
Thanks @dmrvb!  Don't worry, there's no chance of me giving up at this stage :)

As of BETA 132, BorderStyle property is now be read-write, and Control class is (once again) exposed.
